MaskRCNN/data.py

```diff
@@ -478,6 +478,115 @@ def preprocess(roidb):
     return ds
 
 
+def get_viz_dataflow(name):
+    """
+    Return a training dataflow. Each datapoint consists of the following:
+
+    An image: (h, w, 3),
+
+    1 or more pairs of (anchor_labels, anchor_boxes):
+    anchor_labels: (h', w', NA)
+    anchor_boxes: (h', w', NA, 4)
+
+    gt_boxes: (N, 4)
+    gt_labels: (N,)
+
+    If MODE_MASK, gt_masks: (N, h, w)
+    """
+
+    roidbs = DetectionDataset().load_viz_roidbs(name)
```

Comment:
If the only difference between `get_viz_dataflow` and `get_train_dataflow` is the loading roidb part, we better merge these two functions together.

Reply:
Ideally, I'd like to merge these two and the eval dataflow, since they all only have minor differences. I'd rather do that in a separate pull request though, because I don't want to risk breaking anything in the get_train_dataflow.

Reply:
The eval data flow is quite different but we do can merge the get_viz_dataflow and get_train_dataflow. 
You are right we can have a different PR for that.